releasenotes/notes/fixed-ssl-settings-for-addition-endpoints-4e63b4271b21ff17.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixed Logs Agent additional endpoints to respect their
+    logs_no_ssl setting.
+other:
+  - |
+    Added warning for upcoming breaking change to logs_no_ssl
```

Comment:
I'd maybe reword this to something like:

`Added a warning when ``logs_no_ssl`` is set and the url contains an http or https prefix. ``logs_no_ssl`` will take precedence over the prefix in a future agent version.`